alpine/Dockerfile

```diff
@@ -1,69 +1,85 @@
-ARG PHP_IMAGE_TAG=7.2-fpm-alpine
-FROM php:${PHP_IMAGE_TAG}
+ARG PHP_VERSION=7.2
+ARG INVOICENINJA_VERSION
 
-LABEL maintainer="Samuel Laulhau <sam@lalop.co>"
+FROM php:${PHP_VERSION}-fpm-alpine
+
+LABEL maintainer="Samuel Laulhau <sam@lalop.co>,  Holger Lösken <holger.loesken@codedge.de>"
 
 #####
 # SYSTEM REQUIREMENT
 #####
-ENV PHANTOMJS phantomjs-2.1.1-linux-x86_64
-RUN apk update \
- && apk add --no-cache git gmp-dev freetype-dev libjpeg-turbo-dev \
-            coreutils chrpath fontconfig libpng-dev
-
-RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
-    && docker-php-ext-configure gmp \
-    && docker-php-ext-install iconv mbstring pdo pdo_mysql zip gd gmp opcache \
-    && echo "php_admin_value[error_reporting] = E_ALL & ~E_NOTICE & ~E_WARNING & ~E_STRICT & ~E_DEPRECATED">>/usr/local/etc/php-fpm.d/www.conf
+WORKDIR /var/www/app
+
+COPY ./alpine/entrypoint.sh /usr/local/bin/docker-entrypoint
+RUN chmod +x /usr/local/bin/docker-entrypoint
+
+ENV PHANTOMJS="phantomjs-2.1.1-linux-x86_64"
```

Comment:
It doesn't seem to be used actually, we may remove this line

Reply:
Fixed